src/kit.ts

```diff
@@ -502,6 +505,15 @@ export class SessionKit {
             // Call the `afterLogin` hooks that were registered by the LoginPlugins
             for (const hook of context.hooks.afterLogin) await hook(context)
 
+            // Offer session key setup if configured and session does not have one yet
+            if (this.sessionKeyManager && !session.hasSessionKey()) {
+                try {
+                    await session.setupSessionKey()
```

Comment:
What is this for? There's a login plugin already to set one up. 

We don't want this to trigger on transact otherwise.